src/main/shortcutHandler.js

```diff
@@ -0,0 +1,295 @@
+      ['viewChangeFont', 'CmdOrCtrl+.'],
+      ['viewSourceCodeMode', 'CmdOrCtrl+Alt+S'],
+      ['viewTypewriterMode', 'CmdOrCtrl+Alt+T'],
+      ['viewFocusMode', 'CmdOrCtrl+Shift+F'],
+      ['viewToggleSideBar', 'CmdOrCtrl+J'],
+      ['viewToggleTabBar', 'CmdOrCtrl+Alt+B'],
+      ['viewDevToggleDeveloperTools', isOsx ? 'Alt+Command+I' : 'Ctrl+Shift+I'],
+      ['viewDevReload', 'CmdOrCtrl+R']
+    ])
+
+    // fix non-US keyboards
+    this.mnemonics = new Map()
+    this.fixLayout()
+
+    // load user-defined keybindings
+    this.loadLocalKeybindings()
+  }
+
+  fixLayout () {
+    const lang = getKeyboardLanguage()
+    switch (lang) {
+      // fix aidou and inline code
+      case 'ch':
+      case 'de':
+      case 'dk':
+      case 'fi':
+      case 'no':
+      case 'se':
+        this.fixInlineCode()
+
+        if (!isOsx) {
+          this.fixAidou()
+        }
+        break
+
+      // fix aidou only
+      case 'es':
+      case 'hr':
+      case 'it':
+      case 'pl':
+      case 'pt':
+        if (!isOsx) {
+          this.fixAidou()
+        }
+        break
+
+      // custom layouts
+      case 'bg':
+        this.mnemonics.set('CmdOrCtrl+/', 'CmdOrCtrl+8')
+        this.fixInlineCode()
+        break
+    }
+  }
+
+  fixAidou () {
+    this.mnemonics.set('CmdOrCtrl+/', 'CmdOrCtrl+7')
+  }
+
+  // fix dead backquote key on layouts like German
+  fixInlineCode () {
+    this.keys.set('formatInlineCode', 'CmdOrCtrl+Shift+B')
+  }
+
+  loadLocalKeybindings () {
+    if (global.MARKTEXT_SAFE_MODE || !isFile(this.configPath)) {
+      return
+    }
+
+    const json = readJson(this.configPath, true)
+    if (!json || typeof json !== 'object') {
+      log('Invalid keybindings.json configuration.')
+      return
+    }
+
+    // keybindings.json example:
+    // {
+    //   "fileSave": "CmdOrCtrl+S",
+    //   "fileSaveAs": "CmdOrCtrl+Shift+S"
+    // }
+
+    const userAccelerators = new Map()
+    for (const key in json) {
+      if (this.keys.has(key)) {
+        const value = json[key]
+        if (typeof value === 'string' && isAccelerator(value)) {
+          // id / accelerator
+          userAccelerators.set(key, value)
+        }
+      }
+    }
+
+    if (userAccelerators.size === 0) {
+      return
+    }
+
+    // deep clone shortcuts and unset user shortcuts
+    const accelerators = new Map(this.keys)
+    userAccelerators.forEach((value, key) => {
+      accelerators.set(key, '')
+    })
+
+    // check for duplicate shortcuts
+    for (const [userKey, userValue] of userAccelerators) {
+      for (const [key, value] of accelerators) {
+        if (this.equalsAccelerators(value, userValue)) {
+          const err = `Invalid keybindings.json configuration: Duplicate key ${userKey} - ${key}`
+          console.log(err)
+          log(err)
+          return
+        }
+      }
+      accelerators.set(userKey, userValue)
+    }
+
+    // update key bindings
+    this.keys = accelerators
+  }
+
+  get (id) {
```

Comment:
`get` is a `keyword` of JavaScript, it used to declare `get` and `set` properties. So I recommend not use this word as function name. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get